extendr-api/src/graphics/mod.rs

```diff
@@ -405,7 +406,7 @@ impl Context {
     }
 
     pub(crate) fn context(&self) -> pGEcontext {
-        unsafe { std::mem::transmute(&self.context) }
+        &self.context as *const extendr_ffi::R_GE_gcontext as *mut extendr_ffi::R_GE_gcontext
```

Comment:
We should remove graphics anyways, but this casting can be done more succinctly by using https://doc.rust-lang.org/stable/std/primitive.pointer.html#method.cast-1 and friends.